docs/spec/authentic-consistent.en-US.md

```
@@ -78,4 +78,4 @@ title: 真实一致
 - **声音音色**
 - **行为动作**
 
-![](https://mdn.alipayobjects.com/huamei_iwk9zp/afts/img/A*0MTlRYAWlCQAAAAAAAAAAAAADgCCAQ/fmt.webp)
+![](https://mdn.alipayobjects.com/huamei_iwk9zp/afts/img/A*Ao8DSZhMfIAAAAAAAAAAAAAADgCCAQ/original)
```

Markdownlint (0.35.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)